QHyper/solvers/solver.py

```diff
@@ -0,0 +1,10 @@
+from abc import ABC, abstractmethod
+from typing import Any
+
+
+class Solver(ABC):
+    """Interface for solvers"""
+
+    @abstractmethod
+    def solve() -> Any:
```

Comment:
I'm wondering - doesn't an abstract method also need to have "self" as a parameter?

Reply:
Yea, you are right!